src/Microsoft.ML.AutoML/TaskKind.cs

```diff
@@ -9,5 +9,6 @@ internal enum TaskKind
         BinaryClassification,
         MulticlassClassification,
         Regression,
+        Recommendation
```

Comment:
Please add trail comma :)

Reply:
To keep the code style in format

Reply:
note: I could change but this is cherry pick though. it is without trail comma in master branch as well.

Reply:
This is code in `master`. We shouldn't be changing it here. If we want a change like that, make it in the `master` branch.

Honestly, the only thing this branch should be used for is Code Gen and mlnet.csproj going forward. Until they are merged into `master`. Then this branch should go away.